ds_judgements_public_ui/templates/includes/judgment_text_toolbar.html

```diff
@@ -1,10 +1,10 @@
-{% load i18n %}
+{% load i18n document_utils %}
 <div class="judgment-toolbar">
   <div class="judgment-toolbar__container">
     {% if context.document_type == "press_summary" %}
       <p class="judgment-toolbar__press-summary-title">Press Summary</p>
     {% endif %}
-    <h1 class="judgment-toolbar__title">{{ context.judgment_title }}</h1>
+    <h1 class="judgment-toolbar__title">{{ context.page_title|get_title_to_display_in_html:context.document_type }}</h1>
```

Comment:
For PS with name Press Summary of Judgment A. heading of the html should be showing "Judgment A" as we have the press summary label above to show it is a PS. Not really part of the breadcrumbs PR, but since the logic to strip the prefix had to come somewhere, it made sense to fix this as we are here.

Reply:
I think I still need a test for it though

Reply:
have moved to https://github.com/nationalarchives/ds-caselaw-public-ui/pull/805